instrumentation/opentelemetry-instrumentation-urllib3/src/opentelemetry/instrumentation/urllib3/__init__.py

```diff
@@ -151,32 +155,16 @@ class RequestInfo:
     # The type annotations here come from ``HTTPConnectionPool.urlopen()``.
     method: str
     url: str
-    headers: typing.Optional[typing.Mapping[str, str]]
-    body: typing.Union[
-        bytes, typing.IO[typing.Any], typing.Iterable[bytes], str, None
-    ]
-
-
-_UrlFilterT = typing.Optional[typing.Callable[[str], str]]
-_RequestHookT = typing.Optional[
-    typing.Callable[
-        [
-            Span,
-            urllib3.connectionpool.HTTPConnectionPool,
-            RequestInfo,
-        ],
-        None,
-    ]
+    headers: Mapping[str, str] | None
+    body: bytes | typing.IO[typing.Any] | typing.Iterable[bytes] | str | None
+
+
+_UrlFilterT = Optional[Callable[[str], str]]
```

Comment:
Is the Optional kept on purpose?

Reply:
Yep. You can't do the `|` at runtime on Python 3.8 and 3.9.

I can do the following instead:

```suggestion
_UrlFilterT: TypeAlias = 'Callable[[str], str] | None`
```

Both are equivalent. But you need to say it's a `TypeAlias`, otherwise it's interpreted as string.

Reply:
Thanks for the explanation